src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,41 @@
+package duke.task;
+
+import duke.exception.DateFormatException;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+/**
+ * Represents a task with a user-defined start and end time
+ */
+public class Event extends Task {
+    LocalDateTime startTime;
+    LocalDateTime endTime;
+    public Event(String name, String startTime, String endTime) throws DateFormatException {
+        super(name, "E");
+        String[] startString = startTime.split("/");
```

Comment:
For better clarity, how about renaming variables like startString and endString to something like startDateComponents and endDateComponents? It makes understanding the code much easier!